src/olcs/FeatureConverter.js

```diff
@@ -5,6 +5,7 @@ import olGeomGeometry from 'ol/geom/Geometry.js';
 import olStyleIcon from 'ol/style/Icon.js';
 import olSourceVector from 'ol/source/Vector.js';
 import olSourceCluster from 'ol/source/Cluster.js';
+import {circular} from 'ol/geom/Polygon.js';
```

Comment:
Please store it as `olCreateCircularPolygon`, so that it is clear at first glance what the function does and from which library the function comes.

Reply:
sure thing, changed in 7b01d9752dc7ba7edf00832dd6ae39294945363f